### ISSUE TITLE
Support DNS over quic?

### ISSUE THREAD

**Issue by imgk** [mr-karan/doggo#32]
Hi, many thanks for this wonderful tool.

Is there any plan to support DNS over quic?

**Comment by mr-karan** [comment]
Hey. It's already supported

```bash
doggo @quic://dns.adguard.com mrkaran.dev
NAME            TYPE    CLASS   TTL     ADDRESS         NAMESERVER          
mrkaran.dev.    A       IN      30s     3.67.255.218    dns.adguard.com:784
mrkaran.dev.    A       IN      30s     3.67.153.12     dns.adguard.com:784
```

**Comment by imgk** [comment]
I downloaded `v0.4.1` which does not support DNS over quic. Sorry.

**Comment by mr-karan** [comment]
@imgk Ah my bad. I didn't notice that.

Just released v0.4.2 that comes with QUIC support: https://github.com/mr-karan/doggo/releases/tag/v0.4.2 :)
